code/computational_geometry/src/ellipse_drawing/Ellipse.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include <graphics.h>
+
+using namespace std;
+
+void ellipsesymm(int x,int y,int c1,int c2){
+    putpixel(x+c1,y+c2,15);
+    putpixel(-x+c1,y+c2,15);
+    putpixel(x+c1,-y+c2,15);
+    putpixel(-x+c1,-y+c2,15);
+}
+
+void myellipse(int a,int b,int c1,int c2){
+
+    setcolor(BLUE);
+    line(300,0,300,479);

```

Comment:
Take user input

Reply:
I have made the required changes. I am taking the values of major and minor axis as user input.